.github/workflows/release.yml

```diff
@@ -26,14 +32,16 @@ jobs:
           fi
           echo "::set-env name=PIPER_version::$NEXT_VERSION"
 
-      - name: Build, test and push
+      - name: Build and push
         run: |
           echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }} .
-          docker build -t ppiper/node-browsers:${{ env.PIPER_version }}-node12 node12
+          docker build --build-arg=BASE_IMAGE_TAG=${{ matrix.base-image-tag }} --tag ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }} .
+          docker push ppiper/node-browsers:${{ env.PIPER_version }}-${{ matrix.base-image-tag }}
+      - name: Tag and push node 10 image
+        if: ${{ matrix.base-image-tag == 10-buster }}
+        run: |
+          docker tag ppiper/node-browsers:10-buster ppiper/node-browsers:${{ env.PIPER_version }}
```

Comment:
Is the docker login from the previous step still valid here?

Reply:
I'm pretty sure it is, but I'll test in another repo.